spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/DatabaseInitializerMode.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2012-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.autoconfigure;
+
+/**
+ * Supported {@link AbstractDatabaseInitializer database initializer} modes.
+ *
+ * @author Vedran Pavic
+ * @since 2.0.0
+ */
+public enum DatabaseInitializerMode {
```

Comment:
While choosing the target package for `DatabaseInitializerMode` it seemed to me that it would be natural to place it next to `AbstractDatabaseInitializer` which is in `org.springframework.boot.autoconfigure`. I'm not sure what's the best place for those two.